test/fixtures/verbose/index.js

```diff
@@ -1 +1 @@
-setInterval(() => console.log(Math.random()), 100)
+setInterval(() => console.log('[31m' + Math.random() + '\n'), 200)
```

Comment:
Does this have an escape cgaracter in it? If so, can you put it in the string with a backslash escape so it’s easier to read?

Reply:
Sorry, I don't really get what you mean. Nothing in the string needs escaping.